biorxiv/pca_association_experiment/biorxiv_pca_plots.py

```diff
@@ -0,0 +1,639 @@
+
+# ### PCA1 vs PCA 3
+
+# In[11]:
+
+
+display_clouds(
+    'output/word_pca_similarity/figures/pca_01_cossim_word_cloud.png',
+    'output/word_pca_similarity/figures/pca_03_cossim_word_cloud.png'
+)
+
+
+# These word clouds depict the following concepts: quantitative biology vs molecular biology (left) and disease vs sequencing (right)
+
+# In[12]:
+
+
+g = (
+    p9.ggplot(pca_df)
+    + p9.aes(x="pca1", y="pca3", color="factor(category)")
+    + p9.geom_point()
+    + p9.labs(
+        title="PCA of BioRxiv (Word Dim: 300)",
+        color="Article Category"
+    )
+)
+g.save("output/pca_plots/pca01_v_pca03.png", dpi=500)
+print(g)
+
+
+# Take note that pca3 clusters sequencing based papers on the negative axis (down) while disease papers are on the positive axis (up). When plotted against PCA1 it looks like epidemiology papers dominate the top right quadrant, while bottom right quadrant contains bioinformatics papers. This makes sense as many bioinformatic papers consist of some sort of sequencing technologies, while epidemiology is focused on measuring disease and its progression. Both take quantitative views, which is why they are on the positive size of PCA1.
+
+# ### PCA2 vs PCA3
+
+# In[13]:
+
+
+display_clouds(
+    'output/word_pca_similarity/figures/pca_02_cossim_word_cloud.png',
+    'output/word_pca_similarity/figures/pca_03_cossim_word_cloud.png'
+)
+
+
+# These word clouds depict the following concepts: neuroscience to genomics (left) and disease vs sequencing (right)
+
+# In[14]:
+
+
+g = (
+    p9.ggplot(pca_df)
+    + p9.aes(x="pca2", y="pca3", color="factor(category)")
+    + p9.geom_point()
+    + p9.labs(
+        title="PCA of BioRxiv (Word Dim: 300)",
+        color="Article Category"
+    )
+)
+g.save("output/pca_plots/pca02_v_pca03.png", dpi=500)
+print(g)
+
+
+# Note that bottom right quadrant comprises of mainly bioinformatics papers, which makes sense given that quadrant represents sequencing and genomics related papers (hence bioinformatics). The bottom left quadrant contains papers that have sequencing terms mentioned, but are more related to neuroscience than genomics (thats what forms the biophysics clusters). The top left are papers that relate to neuroscience and focused on disease while top right are genomics related papers that focus on disease.
+
+# ### PCA3 vs PCA5 
+
+# In[15]:
+
+
+display_clouds(
+    'output/word_pca_similarity/figures/pca_03_cossim_word_cloud.png',
+    'output/word_pca_similarity/figures/pca_05_cossim_word_cloud.png'
+)
+
+
+# These word clouds depict the following concepts: sequencing vs disease (left) and RNA-seq vs evolutionary biology (right)
+
+# In[16]:
+
+
+g = (
+    p9.ggplot(pca_df)
+    + p9.aes(x="pca3", y="pca5", color="factor(category)")
+    + p9.geom_point()
+    + p9.labs(
+        title="PCA of BioRxiv (Word Dim: 300)",
+        color="Article Category"
+    )
+)
+g.save("output/pca_plots/pca03_v_pca05.png", dpi=500)
```

Comment:
I think Casey's comment would help with the interpretations of the figures. Personally I had to flip several times to remember what each axis of the UMAP plots were so I think someway to mark the axis would be good